js/jquery.fileupload-angular.js

```diff
@@ -24,6 +24,17 @@
             './jquery.fileupload-video',
             './jquery.fileupload-validate'
         ], factory);
+    } else if (typeof exports === 'object') {
+        // Node/CommonJS:
+        factory(
+            require('jquery'),
+            require('angular'),
+            require('./jquery.fileupload'),
```

Comment:
Is it correct that fileupload-angular depends on the process plugin to implicitly import the main jQuery plugin? Anyway, I have removed it for now.